### PR TITLE
Switches to RIALTO person subtypes and adds to person solr index.

### PR DESCRIPTION
**Pull request by justinlittman**
refs sul-dlss-labs/rialto-webapp#113

### CODE REVIEW

**Comment by justinlittman** [comment]
Need to rebase after #132 is merged.

**Comment by jcoyne** [comment]
#132 is merged. Ready to be rebased.

**Comment by justinlittman** [comment]
@jcoyne Rebased and ready for review.

File: models/person.go [hunk 1]
```diff
@@ -86,3 +83,12 @@ func (c *Person) SetCountriesInfo(results *sparql.Results) {
 		c.Countries = append(c.Countries, country)
 	}
 }
+
+// SetPersonSubtypesInfo adds subtypes to a person from sparql results
+func (c *Person) SetPersonSubtypesInfo(results *sparql.Results) {
+	solutions := results.Solutions()
+	for _, solution := range solutions {
+		subtype := &Labeled{solution["subtype"].String(), solution["subtype"].String()[31:]}
```
Comment (jcoyne) [inline models/person.go]: This doesn't seem obvious, so can you leave a comment about why you're taking a slice from the end of this string.

File: transform/person_indexer.go [hunk 2]
```diff
@@ -34,9 +34,12 @@ func (m *PersonIndexer) Index(resource *models.Person, doc solr.Document) solr.D
 	// 4. institution
 	doc.Set("institution_label_ssim", m.retrieveLabels(resource.InstitutionOrgs))
 
-	// 5. institution
+	// 5. countries
 	doc.Set("countries_label_ssim", m.retrieveLabels(resource.Countries))
 
+	// 6. subtype
+	doc.Set("person_subtype_ssi", m.retrieveLabels(resource.Subtypes))
```
Comment (mjgiarlo) [inline transform/person_indexer.go]: If subtypes is multi-valued, should this be `_ssim`?

Comment (justinlittman) [reply transform/person_indexer.go]: Deferring to your solr expertise.

Comment (justinlittman) [reply transform/person_indexer.go]: Fixed.

Comment (mjgiarlo) [reply transform/person_indexer.go]: OK!

**Pull request merged**